homeassistant/components/tplink/light.py

```diff
@@ -246,13 +242,17 @@ def update(self):
             if not self._light_features:
                 self._light_features = self._get_light_features_retry()
             self._light_state = self._get_light_state_retry()
-            self._is_available = True
+            self._is_ready = True
+
         except (SmartDeviceException, OSError) as ex:
-            if self._is_available:
-                _LOGGER.warning(
-                    "Could not read data for %s: %s", self.smartbulb.host, ex
-                )
-            self._is_available = False
+            _LOGGER.warning(
+                "Attempt %s - retrying in %s seconds for %s|%s due to: %s",
+                update_attempt,
+                SLEEP_TIME,
+                self._light_features.host,
+                self._light_features.alias,
+                ex,
+            )
```

Comment:
This could get noisy ... maybe rephrase a bit and only log if `update_attempt == 0` ?

Reply:
Makes sense, added to next commit